src/redux/reducers/player.js

```diff
@@ -37,6 +32,8 @@ const player = (state = INNITAL_STATE, action) => {
     };
   case ADD_GRAVATAR:
     return { ...state, gravatar: action.payload };
+  case UPDATE_SCORE:
+    return { ...state, ...action.payload };
```

Comment:
Talvez, pra deixar mais nítido qual chave foi alterada, colocar a chave sendo reatribuída ao invés de usar spread, oq vc acha? Assim pouparia o trabalho de navegar até a action e depois até a função para saber oq esta sendo recebido, e assim qual chave da store é alterada, faz sentido?
```suggestion
    return { ...state, score: action.payload };
```

Reply:
Concordo,acho q ficaria mais nitido

Reply:
faz sentido, não tinha pensado nisso, vou alterar e commito denovo